src/e2e.test.ts

```diff
@@ -35,6 +35,14 @@ async function setupDataSourceWithClientCredentials(page: Page, clientId: string
     await page.getByTestId('data-testid Data source settings page Save and Test button').click();
 }
 
+async function setupDataSourceWithClientTags(page: Page, clientTags: string) {
+    await page.getByTestId('data-testid Datasource HTTP settings url').fill('http://trino:8080');
+    await page.locator('#trino-settings-enable-impersonation').click({ force: true });
```

Comment:
I think I already asked, but why `force: true` is required here? I don't know much about Playwright tests.